textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/neextractor/NEExtractorTestConstants.java

```diff
@@ -0,0 +1,151 @@
+        IField[] fields1 = {new TextField("Microsoft is an organization.")};
+        ITuple tuple1 = new DataTuple(SCHEMA_ONE_SENTENCE, fields1);
+        return Arrays.asList(tuple1);
+    }
+
+
+
+    public static List<ITuple> getTest2Tuple() throws ParseException {
+
+
+        IField [] fields1 = {new TextField("Microsoft, Google and Facebook are organizations.")};
+        ITuple tuple1 = new DataTuple(SCHEMA_ONE_SENTENCE, fields1);
+        return Arrays.asList(tuple1);
+    }
+
+    public static List<ITuple> getTest3Tuple() throws ParseException {
+
+        IField [] fields1 = {new TextField("Microsoft,Google and Facebook are organizations and Donald Trump and Barack Obama are persons.")};
+        ITuple tuple1 = new DataTuple(SCHEMA_ONE_SENTENCE, fields1);
+        return Arrays.asList(tuple1);
+    }
+
+    public static List<ITuple> getTest4Tuple() throws ParseException {
+
+        IField [] fields1 = {new TextField("Microsoft,Google and Facebook are organizations."), new TextField(" Donald Trump and Barack Obama are persons")};
+        ITuple tuple1 = new DataTuple(SCHEMA_TWO_SENTENCE, fields1);
+        return Arrays.asList(tuple1);
+    }
+
+
+    public static List<ITuple> getTest1ResultTuples() {
+        List<ITuple> resultList= new ArrayList<>();
+        List<IField> fields = new ArrayList<IField>();
+        List<Span> spanList = new ArrayList<Span>();
+        Span span1 = new Span("sentence_one", 0, 8, "ORGANIZATION", "Microsoft");
+        spanList.add(span1);
+        IField spanField = new ListField<Span>(spanList);
+        fields.add(spanField);
+        ITuple resultTuple = new DataTuple(new Schema(SchemaConstants.SPAN_LIST_ATTRIBUTE), fields.toArray(new IField[fields.size()]));
+        resultList.add(resultTuple);
+        return resultList;
+    }
+
+    public static List<ITuple> getTest2ResultTuples() {
+        List<ITuple> resultList= new ArrayList<>();
+        List<IField> fields = new ArrayList<IField>();
+        List<Span> spanList = new ArrayList<Span>();
+        Span span1 = new Span("sentence_one", 0, 8, "ORGANIZATION", "Microsoft");
+        Span span2 = new Span("sentence_one", 10, 15, "ORGANIZATION", "Google");
+        Span span3 = new Span("sentence_one", 21, 28, "ORGANIZATION", "Facebook");
+        spanList.add(span1);
+        spanList.add(span2);
+        spanList.add(span3);
+        IField spanField = new ListField<Span>(spanList);
+        fields.add(spanField);
+        ITuple resultTuple = new DataTuple(new Schema(SchemaConstants.SPAN_LIST_ATTRIBUTE), fields.toArray(new IField[fields.size()]));
+        resultList.add(resultTuple);
+        return resultList;
+    }
+
+    public static List<ITuple> getTest3ResultTuples() {
+        List<ITuple> resultList= new ArrayList<>();
+
+        List<IField> fields = new ArrayList<IField>();
+        List<Span> spanList = new ArrayList<Span>();
+        Span span1 = new Span("sentence_one", 0, 8, "ORGANIZATION", "Microsoft");
```

Comment:
Since "ORGANIZATION", "PERSON", and other labels are very important, maybe you can define and expose them as constants in the operator itself rather in your test cases only?